ts/lib220.ts

```diff
@@ -552,17 +552,19 @@ export const loadJSONFromURL = loadURLHandler(
     }
   ],
   function(runner: any, response: any) {
-    response.json().then((jsonObj : any) => {
-      runner.continueImmediate({
+    response.json().then((jsonObj: any) => {
+      runner.runner.continueImmediate({
         type: 'normal',
         value: stopifyObjectArrayRecur(jsonObj)
       });
     }).catch(() => {
-      runner.continueImmediate({
-        type: 'exception',
-        stack: [],
-        value: new Error(`JSON file could not be loaded`)
-      });
+      runner.isRunning ?
```

Comment:
This is actually an important design pattern! Perhaps `continueImmediate` should effectively have this check built-in.